fibonacci/README.md

```diff
@@ -0,0 +1,8 @@
+# Fibonacci
+
+Function that returns n'th element of Fibonacci sequence.
```

Comment:
Hmm, your solution actually returns the Fibonacci sequence up to the nth element (not just the nth element). May I suggest you change this line to:

> Create a function that returns the Fibonacci sequence up to the nth element.